src/data/general-cs-quiz.ts

```diff
@@ -72,6 +72,17 @@ const generalCSQuiz = [
     Explanation:
       "Big O notation describes the upper bound. The function n upper bounds the function 2*logn. However, it is not the tightest upper bound, and in general we always specify the tightest upper bound to quantify an algorithm.",
     Link: "https://www.youtube.com/watch?v=RBSGKlAvoiM&list=PLWKjhJtqVAbn5emQ3RRG8gEBqkhf_5vxD&t=5m43s",
+  },
+   {
+    Question:
+      "Which is better Merge sort or Quicksort?",
+    Answer: "Merge sort",
+    Distractor1: "Merge sort",
```

Comment:
distractors should be for wrong answers only

```suggestion
    Distractor1: "testing on speed between the two algorithms has been inclusive on which one is faster",
```